inc/helpers/BexDefines.h

```diff
@@ -30,16 +30,25 @@ namespace finalmq {
 
 static const int INVALID_FD = -1;
 
+}   // namespace finalmq
 
-#if defined(WIN32) || defined(__MINGW32__)
 
+#if defined(WIN32) || defined(__MINGW32__)
 #else
-
 typedef int					SOCKET;
```

Comment:
Well we can using more modern syntax here. 
ussing SOCKET = int;

Reply:
I changed it